spec/numberformat.html

```diff
@@ -157,7 +157,7 @@ <h1>FormatNumericToString ( _intlObject_, _x_ )</h1>
       </p>
 
       <emu-alg>
-        1. If _x_ &lt; 0 or _x_ is *-0*<sub>𝔽</sub>, let _isNegative_ be *true*; else let _isNegative_ be *false*.
+        1. If _x_ &lt; 0<sub>𝔽</sub> or _x_ is *-0*<sub>𝔽</sub>, let _isNegative_ be *true*; else let _isNegative_ be *false*.
```

Comment:
This one is kind of tricky - this operation is generic over both Number and BigInt values, so it's not obviously reasonable to compare against the floating-point 0.

I'd suggest instead:

```suggestion
        1. If ℝ(_x_) &lt; 0 or _x_ is *-0*<sub>𝔽</sub>, let _isNegative_ be *true*; else let _isNegative_ be *false*.
```